.circleci/config.yml

```diff
@@ -1,8 +1,11 @@
 version: 2
 
+init_task: &init_task
+  name: Init git submodules
+  command: make init
 build_task: &build_task
   name: Build
-  command: ./gradlew clean assemble check --stacktrace
+  command: make test
 verify_task: &verify_task
   name: Verify Google Java Format
   command: ./gradlew verGJF
```

Comment:
Maybe later, but good to add a comment that we should move this into makefile and all other steps like this. I would suggest an issue is good for the moment.

Reply:
This is trivial I will move it to make right away.